KinSDK/KinSDK/Core/blockchain/TxBuilder.swift

```diff
@@ -9,13 +9,19 @@
 import Foundation
 import KinUtil
 
-public final class TxBuilder {
+@available(*, deprecated, renamed: "TransactionBuilder")
+public class TxBuilder: TransactionBuilder {
+
+}
+
+// TODO: uncomment final after removing TxBuilder
+public /*final*/ class TransactionBuilder {
```

Comment:
I guess that's available for instantiating from the outside world?
I think it should be avoided and be available in `KinAccount` as `getTransactionBuilder` (see comment there), will be much clearer API for devs this way.